ibc/simapp/types.go

```diff
@@ -0,0 +1,43 @@
+package simapp
```

Comment:
Looks like I may have forgot to remove this file. Is the interface needed? and/or defined elsewhere?

Reply:
nope i'll get rid of it

Reply:
done